numpy/polynomial/hermite.py

```diff
@@ -1460,10 +1460,11 @@ def hermfit(x, y, deg, rcond=None, full=False, w=None):
     --------
     >>> from numpy.polynomial.hermite import hermfit, hermval
     >>> x = np.linspace(-10, 10)
-    >>> err = np.random.randn(len(x))/10
+    >>> rng = np.random.default_rng(seed=123)
+    >>> err = rng.standard_normal(size=(len(x)))/10
```

Comment:
```suggestion
    >>> err = rng.norm(scale=1./10., size=len(x))
```

Reply:
I like the change. The extra parenthesis were awkward. I tried to preserve as much of the original example as I could, rather than thinking about using a different function (which here makes it better). I'll make the change here, and below, and I think there is one more as well.